tests/test-org-ql.el

```diff
@@ -2111,7 +2111,84 @@ with keyword arg NOW in PLIST."
           (it "Refuses to link to non-file-backed buffer"
             (expect (var-after-link-save-open 'org-ql-view-buffers-files link-buffer query
                                               :buffer link-buffer)
-                    :to-throw 'user-error '("Views that search non-file-backed buffers can’t be linked to"))))))
+                    :to-throw 'user-error '("Views that search non-file-backed buffers can’t be linked to"))))
+        (describe "Completion for Files/buffers"
+          (describe "Contracting org-ql-view-buffers-files"
+            (it "org-agenda-files from list"
+              (spy-on 'org-agenda-files :and-return-value temp-filenames)
+              (expect (org-ql-view--contract-buffers-files temp-filenames) :to-equal "org-agenda-files"))
+            (it "org-directory"
+              (spy-on 'org-ql-search-directories-files :and-return-value temp-filenames)
+              (spy-on 'org-agenda-files :and-return-value '())
+              (expect (org-ql-view--contract-buffers-files temp-filenames) :to-equal temp-filenames))
+            (it "buffer"
+              (with-current-buffer (org-ql-test-data-buffer "data.org")
+                (expect (org-ql-view--contract-buffers-files (current-buffer)) :to-equal "buffer")))
+            (it "org-agenda-files from symbol"
+              (spy-on 'org-agenda-files :and-return-value temp-filenames)
+              (expect (org-ql-view--contract-buffers-files 'org-agenda-files) :to-equal "org-agenda-files")
+              (expect (org-ql-view--contract-buffers-files #'org-agenda-files) :to-equal "org-agenda-files"))
+            (it "Non specific list"
+              (let ((value1 '("a.org" "b.org"))
+                    (value2 'a))
+                (expect (org-ql-view--contract-buffers-files value1) :to-equal value1)
+                (expect (org-ql-view--contract-buffers-files value2) :to-equal value2))))
+          (describe "Expanding org-ql-view-buffers-files"
+            (it "all"
```

Comment:
For my own edification, it would be helpful to add a clarifying comment like `;; all Org buffers`.  :)

Reply:
Adding to descriptions